test-utils/jestSetup.ts

```diff
@@ -1,11 +1,3 @@
 import 'jest-puppeteer';
 
-import appSnapshotSerializer from './appSnapshotSerializer';
-import cssSnapshotSerializer from './cssSnapshotSerializer';
-import htmlSnapshotSerializer from './htmlSnapshotSerializer';
-
-expect.addSnapshotSerializer(appSnapshotSerializer);
-expect.addSnapshotSerializer(cssSnapshotSerializer);
-expect.addSnapshotSerializer(htmlSnapshotSerializer);
-
```

Comment:
These are already configured in jest config.
https://github.com/seek-oss/sku/blob/e011b59e9191309ce4bd07d86f545f9b2bcd78ef/jest.config.js#L8-L12